### DISCUSSION TITLE
stripe_payment vs flutter_stripe

### DISCUSSION THREAD

**Discussion by fabioselau077** [flutter-stripe/flutter_stripe#673] [Q&A]
Hello, I'm migrating an old code where I use the deprecated stripe_payment plugin.
There I do as follows:
1) Retrieve card data by inputs (name, number, expiration and cvv);
2) Create a CreditCard() with this data.
3) Used createTokenWithCard informing the CreditCard() created earlier.
4) After that, I sent the token and other information to my backend to charge Stripe.

I checked in the documentation that the createTokenWithCard() method was replaced by createToken, but I'm having trouble knowing how to do the same steps I did in the other package.

I don't want to use that native Stripe widget (UI), I already have my widget ready to retrieve the card data.

Thanks

**Comment by ldemyanenko** [comment]
I'm facing same issue. It's not clear if it's possible to create a token on card details with flutter_stripe

**Reply by jun-gh** [reply]
Needs the same functionality also
